1-org/modules/centralized-logging/variables.tf

```diff
@@ -0,0 +1,235 @@
+  description = "The filter to apply when exporting logs. Only log entries that match the filter are exported. Default is '' which exports all logs."
+  type        = string
+  default     = ""
+}
+
+variable "include_children" {
+  description = "Only valid if 'organization' or 'folder' is chosen as var.resource_type. Determines whether or not to include children organizations/folders in the sink export. If true, logs associated with child projects are also exported; otherwise only logs relating to the provided organization/folder are included."
+  type        = bool
+  default     = false
+}
+
+variable "exclusions" {
+  description = "(Optional) A list of sink exclusion filters."
+  type = list(object({
+    name        = string,
+    description = string,
+    filter      = string,
+    disabled    = bool
+  }))
+  default = []
+}
+
+variable "bigquery_options" {
+  description = "(Optional) Options that affect sinks exporting data to BigQuery. use_partitioned_tables - (Required) Whether to use BigQuery's partition tables. Applies to logging target type: bigquery."
+  type = object({
+    use_partitioned_tables = bool
+  })
+  default = null
+}
+
+variable "labels" {
+  description = "(Optional) Labels attached to logging resources."
+  type        = map(string)
+  default     = {}
+}
+variable "kms_key_name" {
+  description = "(Optional) ID of a Cloud KMS CryptoKey that will be used to encrypt the logging destination. Applies to logging target types: bigquery, storage, and pubsub."
+  type        = string
+  default     = null
+}
+
+variable "logging_location" {
+  description = "(Optional) The location of the logging destination. Applies to logging target types: bigquery and storage."
+  type        = string
+  default     = "US"
+}
+
+variable "delete_contents_on_destroy" {
+  description = "(Optional) If set to true, delete all contained objects in the logging destination. Applies to logging target types: bigquery and storage."
+  type        = bool
+  default     = false
+}
+
+#----------------------------- #
+# Logbucket specific variables #
+#----------------------------- #
+variable "retention_days" {
+  description = "(Optional) The number of days data should be retained for the log bucket. Applies to logging target type: logbucket."
+  type        = number
+  default     = 30
+}
+
+#----------------------------- #
+# Big Query specific variables #
+#----------------------------- #
+variable "dataset_description" {
+  description = "(Optional) A user-friendly description of the dataset. Applies to logging target type: bigquery."
+  type        = string
+  default     = ""
+}
+
+variable "expiration_days" {
+  description = "(Optional) Table expiration time. If null logs will never be deleted. Applies to logging target type: bigquery."
+  type        = number
+  default     = null
+}
+
+#--------------------------- #
+# Storage specific variables #
+#--------------------------- #
+variable "storage_class" {
```

Comment:
Could we have a top level `${logging_target_type}_options` input? That will help us manage these better.

Reply:
Done